src/views/AuthenticationWrapper/index.tsx

```diff
@@ -15,6 +15,17 @@ type ParentComponentProps = {
 const AuthenticationWrapper = ({ children }: ParentComponentProps) => {
   const history = useHistory();
 
+  const authClient = new OktaAuth({
+    issuer: process.env.REACT_APP_OKTA_ISSUER,
+    clientId: process.env.REACT_APP_OKTA_CLIENT_ID,
+    redirectUri: process.env.REACT_APP_OKTA_REDIRECT_URI,
+    responseType: ['code'],
```

Comment:
Did you try to remove the `responseType` and see if that made a difference regarding auto renewing? Providing the value of `code` is [not shown in the documentation](https://developer.okta.com/docs/guides/sign-into-spa/react/configure-the-sdk/) (or [here](https://developer.okta.com/code/react/okta_react_sign-in_widget/#config)).